src/booking/Orders/Orders.spec.ts

```diff
@@ -168,4 +170,28 @@ describe('Orders', () => {
       1
     )
   })
+
+  test('should get available services for an order', async () => {
+    nock(/(.*)/)
+      .get(`/air/orders/${mockOrder.id}/available_services`)
+      .reply(200, { data: mockServices })
+
+    const response = await new Orders(
+      new Client({ token: 'mockToken' })
+    ).getAvailableServices(mockOrder.id)
+
+    expect(response.data[0].id).toBe('aso_00009UhD4ongolulWd9123')
```

Comment:
How about this, in case that mock id ever changes in the future?
```suggestion
    expect(response.data[0].id).toBe(mockOrder.id)
```